Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM openjdk:11-jdk-slim-buster AS build
```

Comment:
Any reason why you are using `11-jdk` instead of `openjdk:8-jdk-slim-buster` like the reproducible builds?

Reply:
No, I believe I took that from the rskj Dockerfile, but I'll try using version 8 and correct this.